region/norfair/east/Lower Norfair Elevator.json

```diff
@@ -178,6 +122,51 @@
       ],
       "devNote": "60 frames to cross the room, but 40 to and from the elevator, so the other 10 frames are listed in 3->4."
     },
+    {
+      "link": [1, 4],
+      "name": "Come In Shinecharging",
+      "notable": false,
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 13,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "h_canNavigateHeatRooms",
+        "h_LowerNorfairElevatorDownwardFrames",
+        {"heatFrames": 40}
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 130
+        }
+      }
+    },
+    {
+      "link": [2, 2],
+      "name": "Leave with Runway",
+      "requires": [],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 13,
+          "openEnd": 0
+        }
+      }
+    },
+    {
+      "link": [2, 2],
```

Comment:
could add 1->2/2->1 strats for coming in shinecharging/ed leave charged/sparking